plugins/versionpress/src/Api/VersionPressApi.php

```diff
@@ -201,16 +201,15 @@ public function getCommits(WP_REST_Request $request) {
 
         $initialCommitHash = $this->getInitialCommitHash();
 
-        $isChildOfInitialCommit = $this->gitRepository->wasCreatedAfter($commits[0]->getHash(), $initialCommitHash);
         $isFirstCommit = $page === 0;
 
         $result = array();
         foreach ($commits as $commit) {
-            $isChildOfInitialCommit = $isChildOfInitialCommit && ($commit->getHash() !== $initialCommitHash);
+            $isChildOfInitialCommit = $this->gitRepository->wasCreatedAfter($commit->getHash(), $initialCommitHash);
```

Comment:
I think this has a quite huge performance impact. It runs `git log` for every single displayed commit. Is this the only suitable solution?